message_utils.py

```
@@ -230,6 +248,91 @@ async def get_message_context(message: discord.Message, bot: discord.Client) ->
+        if content_clean.endswith(response) and len(original_content) <= 35:
+            return True
+    
+    # For messages 26-50 characters, be more restrictive
+    # Only allow if they contain common positive/reaction words
+    if 26 <= len(original_content) <= 50:
+        # Check if message contains any of the approved short responses
+        for response in short_responses:
+            if response in content_clean:
+                return True
+        # If no approved words found, reject it
+        return False
```

Ruff (0.11.9)

Use `return any(response in content_clean for response in short_responses)` instead of `for` loop

Replace with `return any(response in content_clean for response in short_responses)`

(SIM110)